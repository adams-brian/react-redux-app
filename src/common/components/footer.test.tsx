import { shallow } from 'enzyme';
import * as React from 'react';

import Footer from './footer';

describe("Footer", () => {

  it('renders as expected', () => {
    const component = shallow(<Footer />);
    expect(component).toMatchSnapshot();
  });
  
});
