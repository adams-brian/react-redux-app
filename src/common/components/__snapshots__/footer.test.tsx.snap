// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders as expected 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <span>
      Built with React and Redux
    </span>
  </div>
</footer>
`;
